lib/laa_crime_forms_common/anonymiser.rb

```diff
@@ -0,0 +1,35 @@
+require "json"
+require_relative "validator"
+
+module LaaCrimeFormsCommon
+  class Anonymiser
+    class << self
+      def anonymise(service_type, payload, version: 1)
+        schema = JSON.parse(File.read(Validator.schema_file(service_type, version:)))
```

Comment:
Do we want to add nice error handling to this? eg this blows up with an invalid `service_type` entered; or do we just let the clients of this handle errors?

Reply:
My gut instinct is that if a client is passing in an invalid service_type or version, that's a dev screwing up their code, and I think we want to blow up loudly to make sure it gets noticed straight away.